converter/src/main/java/gov/cms/qpp/conversion/encode/JsonWrapper.java

```diff
@@ -401,7 +402,12 @@ protected int validInteger(String value) {
 	 */
 	protected String validDate(String value) {
 		try {
-			LocalDate thisDate = LocalDate.parse(cleanString(value),  DateTimeFormatter.ofPattern("yyyyMMdd"));
+			String parse = cleanString(value);
```

Comment:
We should update the JavaDocs to this method.

Reply:
From `Validates that the given value conforms to expected date formatting (i.e. yyyyMMdd).`
To `Validates that the given value may be coerced into conformity to expected date formatting (i.e. yyyyMMdd).`
??

Reply:
I just saw this comment, but I already made the change.